packages/runtime/src/account.ts

```diff
@@ -137,6 +156,12 @@ export class StoreAccount implements StoreAccountI {
       throw new Error('Maximum created applications for an account is 10');
     }
 
+    // raise minimum balance
+    // https://developer.algorand.org/docs/features/asc1/stateful/#minimum-balance-requirement-for-a-smart-contract
+    this.minBalance += (
+      0.1e6 + (SSC_KEY_BYTE_SLICE + SSC_VALUE_UINT) * params.globalInts +
```

Comment:
please update it as const. Ex: REQUIRED_ALGO_APPLICATION something similar

Reply:
ok. adding `APPLICATION_BASE_FEE`